### PR TITLE
[new] [author] added TableExport@3.2.5 w/ git autoupdate

### PR DESCRIPTION
**Pull request by clarketm**
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/clarketm/TableExport
 * Official website (optional, not the repository): https://www.clarketravis.com/tableexport/
 * NPM package url (optional): https://www.npmjs.com/package/tableexport
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 21
   - Count of watchers: 3
   - Count of forks: 2
 * NPM download stats (optional):
   - Downloads in the last day: 77
   - Downloads in the last week: 107
   - Downloads in the last month: 488

# Essential checklist
 * [x] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/TableExport/3.2.5/package.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    "tests"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/clarketm/TableExport.git",
+    "basePath": "dist",
+    "files": [
+      "**/*"
+    ]
+  },
```
Comment (PeterDaveHello) [inline ajax/libs/TableExport/3.2.5/package.json]: the ending comma should be removed.

**Comment by PeterDaveHello** [comment]
@clarketm would you please use out `./tools/fixFormat.js` to remove the useless sections in your package.json? Thanks.

**Comment by clarketm** [comment]
It should now be trimmed and adequately structured.

**Comment by PeterDaveHello** [comment]
@clarketm I also sent some PRs to you here: https://github.com/clarketm/TableExport/pulls/PeterDaveHello, please take a look thanks!

**Comment by clarketm** [comment]
I got those PRs merged. Thanks!

**Comment by PeterDaveHello** [comment]
@clarketm You're welcome.

**Pull request merged**